src/Modules/GraphQL/Type/Fields/EnqueuedScript.php

```diff
@@ -24,19 +24,30 @@ public static function get_type_name(): string {
 		return 'EnqueuedScript';
 	}
 
+	/**
+	 * Register fields to the Type.
```

Comment:
Unless you have reason not to inherit the parent, this ensures any API changes you make are intentional.

```suggestion
	 * {@inheritDoc}
```